lib/susedistribution.pm

```diff
@@ -644,6 +644,14 @@ sub activate_console {
             # case the system is still booting (https://bugzilla.novell.com/show_bug.cgi?id=895602)
             # or when using remote consoles which can take some seconds, e.g.
             # just after ssh login
+            my $retry_num = 0;
+            # retry send ctrl-alt-f key since system not response sometimes
+            while (!(check_screen \@tags, 60) && ($retry_num < 3) && !(check_var('VIRSH_VMM_FAMILY', 'hyperv'))) {
+                $retry_num++;
+                record_soft_failure("system not response refer to poo#45575,try send key again");
```

Comment:
Please only use a record_soft_failure to refer to product issues that you expect to be fixed.

Reply:
So, using record_info instead?

Reply:
yes, record_info would work